### DISCUSSION TITLE
Gifs in README

### DISCUSSION THREAD

**Discussion by alexfleury-sb** [goodchemistryco/Tangelo#284] [Ideas]
I was browsing a python package (https://github.com/Kanaries/pygwalker) that I was not aware of and I wanted to get the feel of using it. To my surprise, there were gifs showing how fundamental tasks are done. I think it would be a good idea to showcase the bar minimum for several important tasks in  Tangelo. However, it may be more suitable to GUI applications, so it may be not worth it.

There would be some challenges:
- Rendering still images when rendering html documentation. I don't know if there would be problems.
- Updating the gifs when major changes happen.

**Comment by ValentinS4t1qbit** [comment]
Thanks for sharing ! I have trouble visualizing what fundamental tasks we are talking about, when it comes to code.
I get the appeal for GUI-based stuff, but how would we use it ?

Would we show a GIF of code being typed or run ?

I could see visual content to be helpful if for example it took readers through what we meant by end-to-end workflows, and what it maps to, but that's sounds like quite a production video / animation work.

**Reply by alexfleury-sb** [reply]
For Tangelo, I would also see something like scrolling notebooks of the end-to-end workflow, or something similar. 

Moreover, I think it would add value the `QEMISTCloudConnection`, as we would be able to show rapidly the interface of the QEMIST client while using quantum circuit simulations.

It is indeed a big project on the video production side, but I don't think it is entirely out of our range of skills.

**Comment by ValentinS4t1qbit** [comment]
Came across this, and actually this sounds like a pretty cool thing to revisit.
